packages/payload/src/fields/config/sanitize.ts

```diff
@@ -267,6 +268,11 @@ export const sanitizeFields = async ({
     if (field.type === 'tabs') {
       for (let j = 0; j < field.tabs.length; j++) {
         const tab = field.tabs[j]
+
+        if ('admin' in tab && tab.admin?.condition && !field.id) {
+          throw new MissingTabsId()
```

Comment:
Shouldn't we just generate some kind of `id` for them? I don't think you should have to make your own `id` in your config to use this feature. What do you think?